src/pages/CatalogPage/CatalogPage.js

```diff
@@ -1,93 +1,71 @@
 import styles from "./CatalogPage.module.css";
+
+import { useState } from "react";
+
+import { Container, useMediaQuery } from '@mui/material';
+import Grid from '@mui/material/Unstable_Grid2';
+
+import Advertising from '../../components/Advertising/Advertising'
+import CategoryList from '../../components/CategoryList/CategoryList'
+import CategoryMoreButton from '../../components/CategoryMoreButton/CategoryMoreButton';
 import Card from "../../components/Card/Card";
 
-const productsTest = [
-  {
-    id: 1,
-    title: "Parfum Chanel №5",
-    image: 'http://placekitten.com/200/300',
-    alt: "Chanel №5 Perfume",
-    price: 2550,
-  },
-  {
-    id: 2,
-    title: "Product 2",
-    image: 'http://placekitten.com/200/300',
-    alt: "Product 2",
-    price: 100,
-  },
-  {
-    id: 3,
-    title: "Product 3",
-    image: 'http://placekitten.com/200/300',
-    alt: "Product 3",
-    price: 500,
-  },
-  {
-    id: 4,
-    title: "Product 4",
-    image: 'http://placekitten.com/200/300',
-    alt: "Product 4",
-    price: 750,
-  },
-  {
-    id: 5,
-    title: "Product 5",
-    image: 'http://placekitten.com/200/300',
-    alt: "Product 5",
-    price: 1200,
-  },
-  {
-    id: 2,
-    title: "Product 2",
-    image: 'http://placekitten.com/200/300',
-    alt: "Product 2",
-    price: 100,
-  },
-  {
-    id: 3,
-    title: "Product 3",
-    image: 'http://placekitten.com/200/300',
-    alt: "Product 3",
-    price: 500,
-  },
-  {
-    id: 4,
-    title: "Product 4",
-    image: 'http://placekitten.com/200/300',
-    alt: "Product 4",
-    price: 750,
-  },
-  {
-    id: 5,
-    title: "Product 5",
-    image: 'http://placekitten.com/200/300',
-    alt: "Product 5",
-    price: 1200,
-  },
-];
+import { products, categories } from '../../data'
+
 
 const CatalogPage = () => {
+  const sm = useMediaQuery('(min-width: 480px)');
```

Comment:
This option is more readable and has less chance to make a mistake.
```suggestion
  const sm = useMediaQuery(theme.breakpoints.up('sm'));
```